node/sc/kas/anchor.go

```diff
@@ -132,9 +133,14 @@ func (anchor *Anchor) AnchorBlock(block *types.Block) error {
 	res, err := anchor.sendRequest(payload)
 	if err != nil || res.Code != codeOK {
 		if res != nil {
+			if res.Code == codeAlreadyAnchored {
+				logger.Info("Already anchored a block", "blkNum", block.NumberU64())
+				return nil
+			}
+
 			result, _ := json.MarshalIndent(res, "", "	")
-			logger.Warn(fmt.Sprintf(`AnchorBlock returns below http raw result with the error(%v) :
-%v`, err, string(result)))
+			logger.Warn(fmt.Sprintf(`AnchorBlock returns below http raw result with the error(%v) at the block(%v) :
```

Comment:
Not related this PR, but how about leaving logs with key value pair?

Reply:
Do you mean that result`'s key is missing?

Reply:
`result` is a json marshaled string.

Reply:
I meant `logger.Warn("message", "err", err, "result", result)`.

Reply:
I tried, but it was difficult to print beautifully. :)
I will consider next time.